client/src/main/java/io/pravega/client/state/impl/RevisionedStreamClientImpl.java

```diff
@@ -50,6 +51,9 @@
     @GuardedBy("lock")
     private final SegmentMetadataClient meta;
     private final Serializer<T> serializer;
+    private final Controller controller;
+    private final String currentDelegationToken;
```

Comment:
this could be just `delegationToken` as it does not change.

Reply:
Fixed.